changelogs/changelog.yml

```diff
@@ -1,5 +1,54 @@
+      - Add a naw parameter 'virtual_tpm' in vars/test.yml for creating new VM with a virtual TPM device.
+      - Move network_testbed_setup tasks to the test case playbook when it's required.
+      - Change to use SATA controller by default for VM CDROM devices.
+      minor_changes:
+      - Add support Debian 10.x and 11.x guest OS auto install.
+      - Add common tasks 'esxi_get_hardware_versions.yml', 'esxi_get_guest_ids.yml' and 'esxi_get_guest_config_options.yml' to get VM hardware version and guest config options.
+      - If the OVT installed is not from Photon OS's repository, skip to run perl GOSC testing on it.
+      - Add OS fullname for AlmaLinux and Rocky Linux in check_os_fullname test case.
+      - Add a workaround for supporting create new VM with IDE boot disk controller.
+      - Change to use common task 'esxi_upload_datastore_file.yml' to avoid using delegate to ESXi host.
+      - Add 'gosc_support_matrix.yml' file to filter supported vCenter version for guest OS.
+      - Change to use connection plugin psrp as the default connection method for Windows VM.
+      - Change to exit testing when ovt_verify_install test case failed for Linux testing.
+      - Add checking image approved keyword in vmware.log after VM deploy when secure boot is enabled.
+      - Move check WinBSOD in vmware.log for Windows after guest OS is unreachable.
+      - Add collect VM vmware.log when test case failed.
+      - Add check reported guestID in guest info in Linux and Windows test case check_os_fullname.
+      - Add gather vmw_pvrdma and ptp_vmw driver info in Linux test case check_inbox_driver.
+      - Remove xorriso package from requirement.txt and add it to README.
+      - Remove ignored failed loop task from failed_tasks.log.
+      - Add ignore_unreachable to true in Windows task which is delegate to guest OS.
+      bugfixes:
+      - Fix VM IP address change causing test case memory_hot_add_basic failure issue.
+      - Fix CentOS online repository URL changing issue.
+      - Fix serial port not removed issue after Ubuntu ova deployment.
+      - Fix template openwrt_19.07.2_x86.ova deployment issue on vSphere 6.5.
+      - Fix check_os_fullname test case failure for Windows 11 VM and add ignore known issue.
+      - Fix failure caused by Flatcar distribution name change.
+      - Fix failed to install sg3-utils package in Debian issue.
+      - Ignore known issue for Windows Server 2022 in test case check_quiesce_snapshot.
+      - Fix mounting VMware tools iso failure issue for existing Windows VM with SATA CDROM.
+      - Fix remove PackageKit package from SLED issue.
+      - Fix snapshot is not quiesced snapshot issue for Windows in test case check_quiesce_snapshot.
+      - Fix to use source IP address to ping target in Windows guest.
+      - Fix ping target result is success when host unreachable issue for Windows.
+      - Fix testing exit when VM unreachable in specific task issue for Ubuntu.
```

Comment:
I suggest using past tense to describe these changes could be more proper, like Add -> Added, Fix->Fixed, etc.